src/app/game-state/activity.service.ts

```diff
@@ -237,7 +237,11 @@ export class ActivityService {
     this.currentApprenticeship = properties.currentApprenticeship || ActivityType.Resting;
     this.savedActivityLoop = properties.savedActivityLoop || [];
     this.autoRestUnlocked = properties.autoRestUnlocked || false;
-    this.pauseOnImpossibleFail = properties.pauseOnImpossibleFail || true;
+    if (properties.pauseOnImpossibleFail === undefined){
```

Comment:
Is there some situation when this would be undefined? Seems like we could handle this in the constructor and setProperties to ensure that its defined.